packages/app/src/app/overmind/effects/gql/dashboard/queries.ts

```diff
@@ -324,6 +324,7 @@ export const recentlyAccessedSandboxes: Query<
     me {
       recentlyAccessedSandboxes(limit: $limit, teamId: $teamId) {
         ...sandboxFragmentDashboard
+        lastAccessedAt
```

Comment:
I didn't add the field to the sandbox fragment, but not sure what to do, maybe it's better to keep the queries/types consistent? but then again, we are overfetching in a lot of situations because of the reuse of the same fragment for the sandbox